components/shared/indexIssueDetails/Comments.js

```diff
@@ -21,7 +21,7 @@ function Comment({ comment }) {
     <Card className="mt-2">
       <Card.Header>
         <b>{comment.user.login}</b>
-        <span className="text-muted">commented on {createdAt}</span>
+        <span className="text-muted ms-1">commented on {createdAt}</span>
```

Comment:
add spacing between the username and "commented on.." in activity tab

Reply:
Nice detail!